docs/spec/lightclient/verification.md

```diff
@@ -844,8 +915,139 @@ If on the blockchain the validator set of the block at height
 
 
 
+## Supporting the IBC Relayer
+
+We might need to download and verify headers whose heights are smaller
+than lightStore.LatestVerified.Height. To formalize
+the solution, we start with the following methods for the LightStore
+
+
+```go
+func (ls LightStore) LatestPrevious(height Height) (LightBlock, bool) 
+```
+- Expected postcondition
+  - returns a light block *lb* that satisfies:
+      - *lb* is in lightStore
+      - *lb* is verified and not expired
+	  - *lb.Header.Height < height*
```

Comment:
@romac is the post condition that the header fetched has not experired satisfied in the current implementation?
Or maybe it's handled somewhere else? 🤔 
https://github.com/informalsystems/tendermint-rs/blob/118b4aee254d417446888f4cd53d5c54ac4c871e/light-client/src/store/memory.rs#L65-L71

Reply:
It's checked here: https://github.com/informalsystems/tendermint-rs/blob/118b4aee254d417446888f4cd53d5c54ac4c871e/light-client/src/light_client.rs#L174-L180